graylog2-web-interface/test/wrappedTestingLibrary.jsx

```diff
@@ -1,14 +1,26 @@
-import { render } from '@testing-library/react';
+// @flow strict
+import { type Element } from 'react';
+import { render, type RenderOptionsWithoutCustomQueries } from '@testing-library/react';
```

Comment:
May need to update the import to use the themed version

```suggestion
import { render, type RenderOptionsWithoutCustomQueries } from 'wrappedTestingLibrary';
```